SR2020/2019-11-04-kit-shipping.md

```diff
@@ -0,0 +1,22 @@
+---
+to: Teams who need to send disclaimer / address
+subject: Kit Shipping
+attachments:
+  - https://drive.google.com/a/studentrobotics.org/file/d/1aCHnZ6o8DkLzHef1C2lz_iIkZkOjYP6P/view?usp=sharing
+---
+
+Hello there!
+
+If you're reading this, it's because you were unable to attend kickstart.
```

Comment:
This isn't strictly true -- it's teams who don't have a kit. (There was at least one team for whom just a team leader turned up and who thus do have a kit, despite not attending Kickstart).

I suggest that we reflect that here.

```suggestion
If you're reading this, it's because we were unable to give you a kit at Kickstart.
```

Reply:
For clarity: this only matters if there are teams who did attend a Kickstart, but didn't collect a kit.